applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_interface_element.cpp

```diff
@@ -0,0 +1,1187 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "custom_constitutive/incremental_linear_elastic_interface_law.h"
+#include "custom_constitutive/interface_plane_strain.h"
+#include "custom_constitutive/interface_three_dimensional_surface.h"
+#include "custom_elements/interface_element.h"
+#include "custom_elements/interface_stress_state.h"
+#include "custom_geometries/interface_geometry.h"
+#include "geo_mechanics_application_variables.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+#include "tests/cpp_tests/test_utilities.h"
+
+#include <boost/numeric/ublas/assignment.hpp>
+#include <cstddef>
+
+namespace
+{
+
+using namespace Kratos;
+using LineInterfaceGeometry2D2Plus2Noded     = InterfaceGeometry<Line2D2<Node>>;
+using LineInterfaceGeometry2D3Plus3Noded     = InterfaceGeometry<Line2D3<Node>>;
+using TriangleInterfaceGeometry3D3Plus3Noded = InterfaceGeometry<Triangle3D3<Node>>;
+using TriangleInterfaceGeometry3D6Plus6Noded = InterfaceGeometry<Triangle3D6<Node>>;
+using Interface2D = Line2DInterfaceStressState;
+using Interface3D = SurfaceInterfaceStressState;
+
+PointerVector<Node> CreateNodes()
```

Comment:
Since we have multiple configurations now, it might be nice to make the name of the function more descriptive:
```suggestion
PointerVector<Node> CreateNodesFor2Plus2LineInterfaceGeometry()
```

Reply:
renamed